src/components/oidc/OidcConfig.ts

```diff
@@ -20,11 +21,9 @@ export const OidcConfig: OidcConfiguration = {
 export function getRolesOfUser(oidcUserInfo: object): string[] {
     let availableRoleList: string[] = [];
     if ('urn:zitadel:iam:org:project:roles' in oidcUserInfo) {
-        if (
-            typeof oidcUserInfo['urn:zitadel:iam:org:project:roles'] === 'object' &&
-            oidcUserInfo['urn:zitadel:iam:org:project:roles'] !== null
-        ) {
-            const roleList: string[] = Object.keys(oidcUserInfo['urn:zitadel:iam:org:project:roles']);
+        const grantedRolesObject = oidcUserInfo['urn:zitadel:iam:org:project:roles'];
```

Comment:
can we also move all the keys names of oidcUserInfo to constants file pls?
Pls consider other fields too in this file.

Reply:
Ok.